src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeAnaylsisHelper.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.MSBuild;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.MSIdentity.CodeReaderWriter
+{
+    public class CodeAnaylsisHelper
```

Comment:
>CodeAnaylsisHelper [](http://example.com/codeflow?start=17&length=18)

typo: Analysis

Reply:
fixed and moved. https://github.com/dotnet/Scaffolding/pull/1593